app/views/content/research.md

```diff
@@ -5,6 +5,14 @@ description: |-
 date: "2021-05-17"
```

Comment:
@EllieNodder do you know what this date field does (and why it's showing date it does)? Aside from that all looks good.

Reply:
I think this is the date the research page was first published